src/plugins/embeddable/public/lib/panel/_embeddable_panel.scss

```diff
@@ -29,6 +29,13 @@
   .embPanel__content--fullWidth {
     width: 100%;
   }
+
+  .embPanel__content--error {
+    &:hover {
+      box-shadow: none;
+      transform: none;
+    }
+  }
```

Comment:
If taking my suggestion to use an `EuiEmptyPrompt` with `EuiButton`, these styles will likely no longer be necessary.